### DISCUSSION TITLE
Can I use tarantool for finding euclidean distance of vectors? Like FAISS or milvus?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [tarantool/tarantool#7393] [Q&A]
Can I use tarantool for finding euclidean distance of vectors? Like FAISS or milvus?

**Comment by Totktonada** [comment]
I'm now experienced in this domain, but maybe [RTREE indexes](https://www.tarantool.io/en/doc/latest/book/box/indexes/#rtree-indexes) will suit your needs.
